server/client_server.go

```diff
@@ -60,67 +61,63 @@ func (client jiraServerClient) GetIssueInfo(projectID string) (*ProjectIssueInfo
 	return &issues, response, err
 }
 
-func (client jiraServerClient) GetProjectInfoForPivotJiraVersion(options *jira.GetQueryOptions) (*jira.CreateMetaInfo, *jira.Response, error) {
-	var issueInfo *ProjectIssueInfo
-	var req *http.Request
-
-	projectList, resp, err := client.Jira.Project.ListWithOptions(options)
-	meta := new(jira.CreateMetaInfo)
+func (client jiraServerClient) GetCreateMetaInfoForPivotJiraVersion(api plugin.API, options *jira.GetQueryOptions) (*jira.CreateMetaInfo, *jira.Response, error) {
+	projectID := options.ProjectKeys
+	proj, resp, err := client.Jira.Project.Get(projectID)
+	if err != nil {
+		return nil, resp, errors.Wrap(err, "failed to get project for CreateMetaInfo")
+	}
 
+	issueInfo, resp, err := client.GetIssueInfo(proj.ID)
 	if err != nil {
-		return nil, resp, errors.Wrap(err, "failed to list projects")
+		return nil, resp, errors.Wrap(err, "failed to get create meta info")
 	}
 
-	for _, proj := range *projectList {
-		meta.Expand = proj.Expand
-		issueInfo, resp, err = client.GetIssueInfo(proj.ID)
+	for _, issueType := range issueInfo.Values {
+		apiEndpoint := fmt.Sprintf("%s%s/issuetypes/%s", APIEndpointCreateIssueMeta, proj.ID, issueType.Id)
```

Comment:
We can use goroutines to process the issue types concurrently.

Reply:
actually here we should not use the goroutines as we don't want the API calls concurrently because it could trigger the API limit for a user.